docs/UserGuide.md

```diff
@@ -181,6 +144,92 @@ Expected output (fail):
 Oops! Invalid search expression, please check again.
 ```
 
+### Viewing a fosterer's detail : `view`
+
+Views a fosterer's details by shifting the foster family list view page to a fosterer's profile view page.
+
+Format: `view INDEX`
+
+Parameters:
+* `INDEX`: Index of a fosterer to view their details shown in the list obtained by the find/list command
+
+<div markdown="span" class="alert alert-primary">
+  :exclamation: <b>Important:</b>
+Some commands are not available while on profile view page. 
+The list of available commands are: 
+<ul>
+  <li> <code>help</code> </li>
+  <li> <code>edit</code> </li>
+  <li> <code>exit</code> command to <b>exit the profile view page</b> </li>
+</ul>
+</div> 
+
+Example:
+* `list` followed by `view 2` to view the profile of the 2nd fosterer in the address book.
+
+Expected output (success):
+```agsl
+Viewing the profile of a fosterer...
```

Comment:
Ouput I get now is "Viewing Person: Jerry Tan; Phone: 98765412; Email: jerry123@example.com; Address: Baker street, block 5, #27-01; Housing: HDB; Availability: NotAvailable; Animal name: Dexter; Animal type: current.Cat; Tags: [Urgent]" but if you are planning to change it eventually then can just leave it as such I guess!

Reply:
Changed it, thanks!